components/Search/Item.tsx

```diff
@@ -0,0 +1,90 @@
+import { EnterIcon } from 'components/Icons'
+import Link from 'next/link'
+import removeMarkdown from 'utils/removeMarkdown'
+import titleCase from 'utils/titleCase'
+
+const getHighlight = ({ title, search, description, content }) => {
+  const s = search.toLowerCase()
+
+  if (title.toLowerCase().includes(s)) {
+    return {
+      type: 'title',
+      highlight: title.toLowerCase().indexOf(s),
+    }
+  }
+  if (description.toLowerCase().includes(s)) {
+    return {
+      type: 'description',
+      highlight: description.toLowerCase().indexOf(s),
+    }
+  }
+
+  if (content.toLowerCase().includes(s)) {
+    return {
+      type: 'content',
+      highlight: content.toLowerCase().indexOf(s),
+    }
+  }
+
+  return {}
+}
+
+const Item = ({ title, url, search, multipleLibs, description, content }) => {
+  const { type, highlight } = getHighlight({ search, title, description, content })
+  const name = titleCase(url.split('/')[1].split('-').join(' '))
+
+  return (
+    <Link href={url}>
+      <a className="block no-underline search-item outline-none">
+        <li className={'px-2 py-1'}>
+          <div className="  p-4 py-5 rounded-md bg-gray-100 hover:bg-gray-800 hover:text-gray-200  flex justify-between items-center transition-all">
+            <span>
+              {multipleLibs ? (
+                <span className="text-xs font-light text-gray-500 block pb-1">{name}</span>
+              ) : null}
```

Comment:
I don't believe we have to use ternaries here. `multipleLibs && <span />` should do.

Reply:
Fixed

Reply:
Also cleaned up this file a bunch